crates/isograph_schema/src/isograph_schema.rs

```diff
@@ -185,6 +175,16 @@ pub struct ServerFieldData {
     pub server_objects: Vec<SchemaObject>,
     pub server_scalars: Vec<SchemaScalar>,
     pub defined_types: HashMap<UnvalidatedTypeName, SelectableServerFieldId>,
+
+    // Well known types
+    pub id_type_id: ServerScalarId,
```

Comment:
all of these types implement copy, they might not need to be moved. I don't really have an opinion on where they go, just commenting